README.md

```diff
@@ -1,2 +1,93 @@
 # o2r-guestlister
-An OAuth server implementation to allow offline login with o2r-bouncer 
+
+A Node.js OAuth2 server implementation to allow offline login with o2r-bouncer as part of the [o2r reference-implementation](https://github.com/o2r-project/reference-implementation).
+
+Based on the repository https://github.com/gerges-beshay/oauth2orize-examples and the underlying OAuth2 server implementation [oauth2orize](https://www.npmjs.com/package/oauth2orize).
+
```

Comment:
1. Just single empty lines in markdown, please.
2. Please describe how ORCID is mimicked, i.e. after login this service also returns a payload of user name and ORCID iD.